src/main/java/org/opentestsystem/ap/imrt/common/model/BaseItem.java

```diff
@@ -73,6 +73,12 @@
     @OrderBy("type ASC")
     private List<StandardId> standardIds;
 
+    @NotBlank
+    private String organizationTypeId = StringUtils.EMPTY;
```

Comment:
According to the documentation for this story, this field will be one of four values.  I chose not to enumerate those values because:
1) I don't think it's IMRT's responsibility to enforce what IAT sends in this field
2) IAT will be enforcing those values in the UI; their `Item` on the server-side accepts any string